guidelines/index.html

```diff
@@ -2975,10 +2975,10 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
    
 </dd>
             
-            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">perimeter</dfn></dt>
+            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">外周 (perimeter)</dfn></dt>
 <dd class="new"><p class="change">[New]</p>	
-   <p>continuous line forming the boundary of a shape not including shared pixels, or the <a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="[New]">minimum bounding box</a>, whichever is shortest.</p>
-  <aside class="example" id="example-16"><div class="marker">例</div><p>The perimeter calculation for a 2 CSS pixel perimeter around a rectangle is 4<em>h</em>+4<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. For a 2 CSS pixel perimeter around a circle it is 4𝜋<em>r</em>.</p></aside>
+   <p>共有されたピクセルがない形状の、境界を形成する連続線、又は<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="[New]">最小境界ボックス</a>、のいずれか短い方。</p>
+  <aside class="example" id="example-16"><div class="marker">例</div><p>長方形を囲む 2 CSS ピクセルの外周の算出方法は、4<em>h</em> + 4<em>w</em> であり、<em>h</em> は高さを、<em>w</em> は横幅を表す。円を囲む 2 CSS ピクセルの外周は、4𝜋<em>r</em> である。</p></aside>
```

Comment:
whereは数学なんかでいう「ここで」ですね。
[一般閃光閾値及び赤色閃光閾値 (general flash and red flash thresholds)](https://waic.jp/translations/WCAG21/#dfn-general-flash-and-red-flash-thresholds)ではそのように訳しています。

ここ、長方形にしますかね…。読みやすさでいうと長方形ですが、あわせるなら矩形ですよね…。

```suggestion
  <aside class="example" id="example-16"><div class="marker">例</div><p>長方形を囲む 2 CSS ピクセルの周長の算出方法は、4<em>h</em> + 4<em>w</em> である。ここで、<em>h</em> は高さを、<em>w</em> は横幅を表す。円を囲む 2 CSS ピクセルの周長は、4𝜋<em>r</em> である。</p></aside>
```

Reply:
where を「ここで」、rectangle を「矩形」とすること、異論ありません。
あとは、perimeter を「周長」と訳すかどうか、ですね。

Reply:
こちらの perimeter の意味は「周長」ですね。同じ perimeter という単語ですが、文脈に応じて訳し分けるのがいいかなと。

この注釈に辿り着くまでの流れをメモしておくと、

- 達成基準2.4.13 Focus Appearanceでは "2 CSS pixel thick perimeter" と同等以上の面積が要求される (ここでは面積の話をしていて、この perimeter は「周囲」)
- この perimeter の定義として、用語の perimeter がある。そこではboundary of a shape と minimum bounding box のどちらか短い方が採用されることになっている (ここは先の面積の話を引き継いでいる定義で、この perimeter は「周囲」)
- boundary of a shape と minimum bounding box のうちどちらが短いかを調べる必要があるため、それぞれの長さの計算方法の例として注釈を書いた  (ここでは長さの話をしていて、この perimeter は「周長」)

ということで、ここでは長さの計算の話をしていて、perimeter は周長です。
(なのですが、正直「この注釈いります? 逆にわかりにくくなってませんか?」という感想……)